RInAppMessaging/Classes/Protocols/TaskSchedulable.swift

```diff
@@ -14,10 +14,11 @@ internal protocol TaskSchedulable: AnyObject {
 /// Default implementation.
 extension TaskSchedulable {
     func scheduleWorkItem(milliseconds: Int, wallDeadline: Bool, closure: @escaping () -> Void) {
-
-        scheduledTask?.cancel()
-        scheduledTask = WorkScheduler.scheduleTask(milliseconds: milliseconds,
-                                                   closure: closure,
-                                                   wallDeadline: wallDeadline)
+        DispatchQueue.main.async {
```

Comment:
Why put this on the main thread?

Reply:
It just to prevent resource conflicts when accessing scheduledTask property. The issue was visible sometimes when running tests (EXC_BAD_ACCESS crash)